ajax/libs/slickgrid/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "slickgrid",
+  "version": "2.3.4",
+  "description": "A lightning fast JavaScript grid/spreadsheet",
+  "filename": "slick.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/6pac/SlickGrid.git"
+  },
+  "keywords": [
+    "slickgrid",
+    "grid"
+  ],
+  "author": {
+    "name": "Michael Leibman",
+    "email": "michael.leibman@gmail.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/mleibman/SlickGrid/wiki",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/6pac/SlickGrid.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "lib/jquery.event.*",
+          "lib/firebugx*",
+          "images/*.+(gif|png)",
+          "slick*",
+          "plugins/slick*.+(js|css|map)"
```

Comment:
May I ask what situation should use `*` or `+(js|css|map)`?
Why the files under `images` should use `"images/*.+(gif|png)"`, and why the files under `plugins` should use `"plugins/slick*"`?